tests/test_ordering.py

```diff
@@ -0,0 +1,44 @@
+import re
+from collections import defaultdict
+from collections import OrderedDict
+import sys
+sys.path.append('../user_query_optimizer')
+import optimizer
+import sqlparse
+
+def test_ordering(queries, presto_op):
+    # dictionary from test-query-file -> list of line numbers in that query with an approx optimization
+    correct_ops = {
+        'test-query-1.txt': [0],
+        'test-query-2.txt': [3],
+        'test-query-3.txt': [0],
+        'test-query-4.txt': [0, 4],
+        'test-query-5.txt': [3],
+        'test-query-6.txt': [0], # for some reason, sqlparse not recognizing two statemnets
```

Comment:
Did you file this bug? also s/statemnets/statements

Reply:
yes, filed!